### ISSUE TITLE
when you upload an image in an empty media section for the first time nothing shows up

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes-Front-End#4365]
If you have an empty media section and then upload an image, the front isn't don't a get and grabbing the uploaded image. You have to reload the page via the browser refresh to show that the image is uploaded. It basically looks like something went wrong with the upload. 

If there is already an uploaded past images in the media section, then there is a get command and the new uploaded image appears 

<img width="1147" alt="screen shot 2018-02-16 at 8 03 39 am" src="https://user-images.githubusercontent.com/753325/36322282-cf744e94-12f0-11e8-94b4-c120e3116c6b.png">

upload an image for the first time and it stays like above. 

Hit refresh on the browser:

<img width="1131" alt="screen shot 2018-02-16 at 8 03 47 am" src="https://user-images.githubusercontent.com/753325/36322303-e3d791ca-12f0-11e8-952f-0481a64e79b2.png">

upload a second image and no problem. 

It appeared next the first image without me having to use the browser refresh

<img width="1090" alt="screen shot 2018-02-16 at 8 08 07 am" src="https://user-images.githubusercontent.com/753325/36322369-1d9a0258-12f1-11e8-92b0-eaf770b8b624.png">

The behavior is as expected. I think this important to fix because for new people using the project builders it looks like their image doesn't upload and keep uploading the same image which doesn't appear but has actually uploaded. 

I'm using firefox quantum 58.0.2 for mac on mac high sierra . I expect the behavior is likely the same in other browsers

**Comment by eatyourgreens** [comment]
I've just tried this in Chrome 65 on a Mac (El Capitan). I created a new, empty project and dragged a file into the media area. the display refreshed and showed me my new photo. I deleted that photo, clicked on the big + sign, chose a photo and that uploaded without having to reload the page.

I'll try again in Firefox. This might be a browser bug.

**Comment by eatyourgreens** [comment]
After chatting to @mschwamb about this, we think it might only happen on slow collections. The media area won’t update if the upload times out, so the issue might be one of how we message network errors here.

**Comment by eatyourgreens** [comment]
If the connection to the server drops, there’s no response so no way to know if the upload failed. Hence you having to refresh the page to check if the image was uploaded.

**Comment by eatyourgreens** [comment]
If I drop the network connection during an upload, then an error message is displayed (as expected.)
![A screenshot of the media upload area, showing an error when the network is offline.](https://user-images.githubusercontent.com/59547/47213130-06f1dc80-d392-11e8-8f50-1400515c5ac1.png)

**Comment by yshish** [comment]
It happened to me a few days ago too. I uploaded an image to my new test project for the first time and it didn't appear as uploaded. So I repeated the uploading process. Nothing. Repeated again. Nothing. Then I left the page and came back again to find the same image uploaded there triple times. I removed the first two images and uploaded a different one. With no troubles. It only happened with the first uploading before the uploaded images displayed.

My internet connection speed for download is 44,02 Mbit/s, and 9,94 Mbit/s for upload. If that helps. Working on Chromium/Ubuntu.
